src/NinjaTools.FluentMockServer.API/Models/RequestMatcher.cs

```diff
@@ -0,0 +1,178 @@
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+using System.Text;
+using JetBrains.Annotations;
+using Microsoft.AspNetCore.Http;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Converters;
+
+namespace NinjaTools.FluentMockServer.API.Models
+{
+    [DebuggerDisplay("{DebuggerDisplay()}")]
+    public class RequestMatcher
+    {
+        public string DebuggerDisplay()
+        {
+            var path = Path + QueryString ?? "";
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unused local variable 'path'.](https://app.codacy.com/manual/git_36/NinjaTools.FluentMockServer/pullRequest?prid=4802642)